polkadot/node/network/availability-recovery/src/lib.rs

```diff
@@ -88,8 +97,15 @@ pub enum RecoveryStrategyKind {
 	/// We try the backing group first if PoV size is lower than specified, then fallback to
 	/// validator chunks.
 	BackersFirstIfSizeLower(usize),
+	/// We try the backing group first if PoV size is lower than specified, then fallback to
+	/// systematic chunks.
+	BackersFirstIfSizeLowerThenSystematicChunks(usize),
 	/// We always recover using validator chunks.
 	ChunksAlways,
+	/// First try the backing group. Then systematic chunks.
+	BackersThenSystematicChunks,
+	/// Always recover using systematic chunks, fall back to regular chunks.
+	SystematicChunks,
```

Comment:
tbh, I don't think we need as many recovery strategies, this should be reduced to only one or two eventually

Reply:
I generally agree. However, this helps with unit testing and performance comparison for now

Reply:
I want to keep these for easier unit testing (however, I removed their respective subsystem constructor, so they can only be constructed from tests).